pycaret/internal/pycaret_experiment/time_series_experiment.py

```diff
@@ -2857,12 +2857,18 @@ def predict_model(
             # result = result.join(return_vals[1])
             result = pd.concat(return_vals, axis=1)
             result.columns = ["y_pred", "lower", "upper"]
+            result["within_pi"] = result.apply(
```

Comment:
This looks pretty heavy. I wonder if it's possible to use vector operations for this

Reply:
OK, I have pushed the changes. Please review. Thanks!